storagetransfer/transfer_between_posix.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storagetransfer
+
+// [START storagetransfer_transfer_between_posix]
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	storagetransfer "cloud.google.com/go/storagetransfer/apiv1"
+	storagetransferpb "google.golang.org/genproto/googleapis/storagetransfer/v1"
+)
+
+func transferBetweenPosix(w io.Writer, projectID string, sourceAgentPoolName string, sinkAgentPoolName string, rootDirectory string, destinationDirectory string, bucketName string) (*storagetransferpb.TransferJob, error) {
+	// Your project id
+	// projectId := "my-project-id"
+
+	// The agent pool associated with the POSIX data source. If not provided, defaults to the default agent
```

Comment:
Here and elsewhere-- there's no way to "not provide" a function arg in Go. How would the user get the default values if they want them? Is it enough to pass an empty string, or does something different need to happen in how the request is constructed?

Reply:
It would just be passing in an empty string, or even just not specifying the parameter, either works

Reply:
Gotcha, makes sense. I guess since this is docs inside the function rather than a function godoc, it's fine to phrase it this way.